doc/transform_methods/daun.rst

```diff
@@ -0,0 +1,332 @@
+of broad and narrow Gaussian peaks and its approximations of various degrees is
+shown below:
+
+.. plot:: transform_methods/daun-basis.py
+    :align: center
+
+Here the solid black line is the test function, and the dashed black line is
+its approximation of degree :math:`n`, equal to the sum of the colored basis
+functions.
+
+degree = 0:
+    Rectangular functions produce a stepwise approximation. This is the only
+    approach mentioned in the original article and corresponds to the usual
+    “onion peeling” transform.
+degree = 1:
+    `Triangular functions
+    <https://en.wikipedia.org/wiki/Triangular_function>`__ produce a continuous
+    piecewise linear approximation. Besides being continuous (although not
+    smooth), this also corresponds to how numerical data is usually plotted
+    (with points connected by straight lines), so such plots would faithfully
+    convey the underlying method assumptions.
+degree = 2:
+    Piecewise quadratic functions
+
+    .. math::
+        f_i(r) = \begin{cases}
+            2[r - (i - 1)]^2, & r \in [i - 1,   i - 1/2], \\
+            1 - 2[r - i]^2,   & r \in [i - 1/2, i + 1/2], \\
+            2[r - (i + 1)]^2, & r \in [i + 1/2, i + 1], \\
+            0                 & \text{otherwise}.
+        \end{cases}
+
+    produce a smooth piecewise quadratic approximation. While resembling
+    :ref:`BASEX basis functions <BASEXcomp>` in shape, these are localized
+    within ±1 pixel, sum to unity (although produce oscillations on slopes),
+    and their projections are much faster to compute.
+degree = 3:
+    Combinations of `cubic Hermite basis functions
+    <https://en.wikipedia.org/wiki/Cubic_Hermite_spline#Interpolation_on_a_single_interval>`__
+    produce a cubic-spline approximation (with endpoint derivatives clamped to
+    zero for 2D smoothness). Offers the most accurate representation for
+    sufficiently smooth distributions, but produces ringing artifacts around
+    sharp features, which can result in negative interpolated intensities even
+    for non-negative data points.
+
+(The projections of all these basis functions are calculated as described in
+:ref:`Polynomials`.)
+
+In practice, however, the choice of the basis set has negligible effect on the
+transform results, as can be seen from an example :ref:`below
+<example_degrees>`. Nevertheless, cubic splines might be useful for
+transforming smooth functions, in which case they yield very accurate results.
+
+
+Regularization methods
+^^^^^^^^^^^^^^^^^^^^^^
+
+:math:`L_2` norm
+""""""""""""""""
+
+In addition to the original derivative (difference) Tikhonov regularization,
+PyAbel also implements the usual :math:`L_2` regularization, as in
+:ref:`BASEX`, with the identity matrix :math:`\mathbf I` used instead of
+:math:`\mathbf L` in :eq:`tikhonov`. The results are practically identical to
+the BASEX method, especially with **degree** = 2, except that the basis set is
+computed much faster.
+
+Non-negativity
```

Comment:
I have mixed feelings about this option. It is fundamentally wrong, produces incorrect results for the background intensity, only to give the illusion of smoothing. However, it reproduces the old PC `basex` output, so it may have some use.

Here is the `O2-ANU2048.txt` (larger image size to increase noise) example, note incorrect background between transitions for the `nonneg` option:
![image](https://user-images.githubusercontent.com/10932229/132081696-78a3e8dc-87f5-4982-a404-fc7ca435dd40.png)

Reply:
Sorry, what is fundamentally wrong?

Reply:
The non-negativity option (`reg='nonneg'` parameter) limits the inverse Abel transform to positive values. This option may result in an under correction for low pixel intensities, as shown in the figure above (blue line vs orange). This demonstration is for good quality VMI data, the effect is much more dramatic for lesser quality images, and the effect varies for the particular spectral structure.  "Fundamentally wrong" in that the option results may produce an incorrect spectral profile. The old PC `basex.exe` produced only positive valued inverse Abel transforms (images). 

My apology if I have misunderstood what this option does.

Reply:
I've answered [below](https://github.com/PyAbel/PyAbel/pull/326#issuecomment-915483616).